ospp.vbs: add page (#12666)
tldr-pages/tldr: pages/windows/ospp.vbs.md

# ospp.vbs

> > Install, activate, and manage volume licensed versions of Office products.

Reviewer: ```suggestion
> Install, activate, and manage volume licensed versions of Microsoft Office products.
```

If this program works exclusively with Microsoft Office, then I would mention the name Microsoft at least once in the description.

> > This command may override, deactivate, and/or remove your current volume licensed versions of Office products. Please proceed with caution.

Reviewer 2: ```suggestion
> Note: this command may override, deactivate, and/or remove your current volume of licensed Office product versions, so please proceed cautiously.
```
Maybe, we can update this description and convert it to a note?

> More information: <https://learn.microsoft.com/deployoffice/vlactivation/tools-to-manage-volume-activation-of-office>.

> - Install a product key (replace existing key):

Reviewer 2: ```suggestion
- Install a product key (Note: it replaces the existing key):
```

`cscript ospp.vbs /inpkey:{{product_key}}`

- Uninstall an installed product key with the last five digits of the product key:

> `cscript ospp.vbs /unpkey:{{product_key}}`

Reviewer 3: ```suggestion
`cscript ospp.vbs /unpkey:{{product_key_digits}}`
```
Just in case anyone takes a look at the command first and then read the associated description.

> `cscript ospp.vbs /unpkey:{{last_five_digits_of_the_product_key}}`

Reviewer 2: ```suggestion
`cscript ospp.vbs /unpkey:{{product_key}}`
```
Since it's already specified in the description, I don't think we need to repeat this again.

- Set a KMS host name:

> `cscript ospp.vbs /sethst:{{kms_server_ip_or_host_name}}`

Reviewer 3: ```suggestion
`cscript ospp.vbs /sethst:{{ip|hostname}}`
```

- Set a KMS port:

> `cscript ospp.vbs /setprt:{{port_number}}`

Reviewer 3: ```suggestion
`cscript ospp.vbs /setprt:{{port}}`
```
This is not ambiguous, right?
